lib/messageUtils.ts

```diff
@@ -50,3 +50,11 @@ export function getDashboard( template: string, msgId: string ) : string | undef
 
   return undefined
 }
+
+export function getPreviewLink( message: any ) : string | undefined {
```

Comment:
It seems like this function would never actually return undefined, would it?

If not, can we remove it from the type signature?

Reply:
yep!